pkg/cmd/gen_driftignore_test.go

```diff
@@ -158,3 +143,10 @@ func TestGenDriftIgnoreCmd_InvalidFlags(t *testing.T) {
 		assert.EqualError(t, err, tt.err.Error())
 	}
 }
+
+// The leading comment, "Generated by gen-driftignore..." contains a timestamp,
```

Comment:
I'm good with that for now, but this something I consider hacky.

I consider time like any other dependency in a domain codebase, that should have an interface and be mock-able.
Since the gen-driftignore does not have a clear domain service code let's keep that for the moment. I just wanted to add my 50c here 😅

Reply:
Sure, injecting a fake clock is something I've done a lot before for similar problems. I'm definitely on board with mock-driven TDD. But, it's a question of how much we want to go about refactoring this part of driftctl for this individual PR. Maybe this can almost be considered technical debt? Up to you

Reply:
> But, it's a question of how much we want to go about refactoring this part of driftctl for this individual PR. Maybe this can almost be considered technical debt? Up to you

Of course it is tech debt and it has nothing to do with this PR 🙂 Sorry if that wasn't clear enough in my comment.

Reply:
it was, I think I was just being overly explicit 😅